mijia/src/lib.rs

```diff
@@ -158,7 +158,7 @@ impl MijiaEvent {
 /// let (_, session) = MijiaSession::new().await?;
 ///
 /// // Start scanning for Bluetooth devices, and wait a few seconds for some to be discovered.
-/// session.bt_session.start_discovery().await?;
+/// session.bt_session.start_discovery(&Default::default()).await?;
```

Comment:
do we have anything in a README.md that needs changing?

Reply:
I updated this in two README.md files already, is there anything else I missed?